nursery/persist-via-application-shimming.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: persist via application shimming
+    namespace: persistence/file-system-and-registry
+    authors:
+      - j.j.vannielen@utwente.nl
+    scopes:
+      static: function
+      dynamic: call
+    att&ck:
+      - Persistence::Event Triggered Execution::Application Shimming [T1546.011]
+    references:
+      - https://cloud.google.com/blog/topics/threat-intelligence/fin7-shim-databases-persistence/
+  features:
+    - or:
+      - and:
+        - match: set registry value
+        - string: /Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\InstalledSDB\\/i
```

Comment:
what about the `Custom` entry?

Reply:
We could add that as an optional field. The malware can gain persistence by only overwriting the InstalledSDB of an existing entry

Reply:
ok, if `Custom` is not a indicator by itself, let's leave as is